### ISSUE TITLE
Masternode payout split my coin, invalidating my 10k tx, disabling masternode

### ISSUE THREAD

**Issue by fbastage** [PIVX-Project/PIVX#145]
### Describe the issue
I received masternode payout, but for some reason my 10k masternode transaction inputs were spent (split into 2 chunks, back to me; DJ74owepQJXq1WTvY5wws31pVAVS91C6B1).  This disabled my masternode.

see: http://www.presstab.pw/phpexplorer/PIVX/tx.php?tx=1b159d74f0a12150e6bc48a84c586d9fa00d7ef6d6236887abdf1316a97af94b

Transaction ID:	1b159d74f0a12150e6bc48a84c586d9fa00d7ef6d6236887abdf1316a97af94b
Included in Block:	613007
Transaction Time:	23 Apr 2017 23:11:03
Input 0 :	10000 is sent from DJ74owepQJXq1WTvY5wws31pVAVS91C6B1
Input Source: b98dd30d4489819c830d8181275430b670e586390873c5ee5328eeec841a81f9
Output 0 :	CoinStake
Output 1 :	5004.49 sent to DJ74owepQJXq1WTvY5wws31pVAVS91C6B1
Output 2 :	4999.55998518 sent to DJ74owepQJXq1WTvY5wws31pVAVS91C6B1

### Can you reliably reproduce the issue?
no. so far this is a one-off event.

### Expected behavior
My masternode 10k tx should not be split.  No one other than me should be able to 'spend' my coins.

### Actual behavior
my coins were spent, splitting them into 2 chunks.  this caused my masternode to stop operating as a masternode.  (I am in the process of sending back to myself in a 10k chunk to re-start the masternode.)

### What version of PIVX Core are you using?
2.2.0  pivxd/cli on linux for the masternode (Pivx Core Daemon version v2.2.0; Pivx Core RPC client version v2.2.0)
2.2.0 qt on windows for the controller wallet (PIVX Core version v2.2.0 (64-bit))

**Comment by presstab** [comment]
Was this actually a valid masternode? Your 10k coins were staked. Usually when you have your masternode up and running, the 10k are locked and kept away from the staking process.

**Comment by fbastage** [comment]
@presstab ahh, interesting.

what happened is this:  I needed to upgrade and restart a different masternode, so I 'temporarily' set up my wallet locally to start the other masternode.  I did not enter both masternodes in the masternode.conf.  I did enable staking on this wallet, and then left it running.  

so in this 2nd wallet, the 10k for the masternode in question (in the above issue as posted) was not locked in masternode, and was staking.

so it seems you are right that these coins were set up to stake, which I suppose explains what happened.

I'll definitely avoid this in the future.

we can close this issue.
